packages/turbo-workspace-convert/src/cli.ts

```diff
@@ -0,0 +1,187 @@
+
+// find all available package managers
+const availablePackageManagers: Record<
+  PackageManagers,
+  PackageManagerAvailable
+> = {
+  yarn: isYarnAvailable(),
+  pnpm: isPnpmAvailable(),
+  npm: isNpmAvailable(),
+};
+
+function isPackageManagerDisabled({
+  packageManager,
+  currentWorkspaceManger,
+}: {
+  packageManager: PackageManagers;
+  currentWorkspaceManger: PackageManagers;
+}) {
+  if (currentWorkspaceManger === packageManager) {
+    return "already in use";
+  }
+
+  if (!availablePackageManagers[packageManager].available) {
+    return "not installed";
+  }
+
+  return false;
+}
+
+interface Answers {
+  packageManager: PackageManagers;
+}
+
+const help = `
+  Usage:
+    $ npx @turbo/workspace-convert [flags...] [<dir>]
+
+  If <dir> is not provided, you will be prompted for it.
+
+  Flags:
+    --npm           Switch to npm workspaces
+    --pnpm          Switch to pnpm workspaces
+    --yarn          Switch to yarn workspaces
+    --install       Convert lock files and install dependencies
+    --dry           Run without making any modifications
+    --summary       Show a summary of the workspace at <dir>, including the detected package
+                    manager, all workspaces, and their locations
+    --help, -h      Show this help message
+    --version, -v   Show the version of this script
+`;
+
+run().catch((err: Error) => {
+  if (err?.cause !== "expected") {
+    console.log("Unexpected error - aborting");
+    console.error(err.message || err);
+    process.exit(1);
+  }
+  console.log();
+  console.error(err.message || err);
+  process.exit(0);
+});
+
+async function run() {
+  let { input, flags, showHelp, showVersion } = meow(help, {
+    booleanDefault: undefined,
+    flags: {
+      npm: { type: "boolean", default: false },
+      pnpm: { type: "boolean", default: false },
+      yarn: { type: "boolean", default: false },
+      install: { type: "boolean" },
+      dry: { type: "boolean", default: false },
+      summary: { type: "boolean", default: false },
+      help: { type: "boolean", default: false, alias: "h" },
+      version: { type: "boolean", default: false, alias: "v" },
+    },
+  });
+
+  if (flags.help) showHelp();
+  if (flags.version) showVersion();
+  const logger = new Logger({ dry: flags.dry, interactive: true });
+
+  logger.hero();
+  await new Promise((resolve) => setTimeout(resolve, 500));
+  logger.header("Welcome, let's convert your project.");
+  logger.blankLine();
+  // Figure out the app directory
+  let projectDir = path.resolve(
+    process.cwd(),
+    input.length > 0
+      ? input[0]
+      : (
+          await inquirer.prompt<{ dir: string }>([
+            {
+              type: "input",
+              name: "dir",
+              message: "Where is the root of your monorepo?",
+              default: ".",
+            },
+          ])
+        ).dir
+  );
+
+  let shouldInstall = flags.install;
+  if (shouldInstall === undefined && !flags.dry) {
```

Comment:
Make the variable state always map to the meaning.

```suggestion
  let shouldInstall = flags.dry === true ? false : flags.install;
  if (shouldInstall === undefined) {
```